libc/newhdrgen/yaml/pthread.yaml

```diff
@@ -13,33 +13,8 @@ types:
   - type_name: __pthread_start_t
   - type_name: __pthread_once_func_t
   - type_name: __atfork_callback_t
-enums:
-  - name: PTHREAD_CREATE_JOINABLE
```

Comment:
why did these get removed?

Reply:
Was already in the h.def prior so we were duplicating enums.